### PR TITLE
Created empty JS Project

### PR DESCRIPTION
**Pull request by ihrynkiv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# *Tic Tac Toe Game*
```
Comment (andrii-s-shpitser-lpnu-ua) [inline README.md:1]: add more details (HW i-fase, data driven type)

Comment (ihrynkiv) [reply README.md:1]: updated README

**Comment by andrii-s-shpitser-lpnu-ua** [comment]
delete main branch and do develop as default branch

**Comment by ihrynkiv** [comment]
> delete main branch and do develop as default branch

@andrii-s-shpitser-lpnu-ua deleted main branch, make develop as default branch

**Review by andrii-s-shpitser-lpnu-ua — APPROVED** [review]

**Pull request merged**